src/module/skb/skb.rs

```diff
@@ -54,23 +58,29 @@ impl Collector for SkbModule {
         // First, get the cli parameters.
         let args = cli.get_section::<SkbCollectorArgs>(ModuleId::Skb)?;
 
-        let mut sections: u64 = 1 << SECTION_PACKET;
+        // Default list of sections.
+        let mut sections: u64 = 1 << SECTION_PACKET
+            | 1 << SECTION_ETH  // FIXME.
+            | 1 << SECTION_ARP
+            | 1 << SECTION_IPV4
+            | 1 << SECTION_IPV6
+            | 1 << SECTION_TCP
+            | 1 << SECTION_UDP
+            | 1 << SECTION_ICMP;
+
         for category in args.skb_sections.iter() {
             match category.as_str() {
                 "all" => sections |= !0_u64,
-                "eth" => sections |= 1 << SECTION_ETH,
-                "arp" => sections |= 1 << SECTION_ARP,
-                "ip" => sections |= 1 << SECTION_IPV4 | 1 << SECTION_IPV6,
-                "tcp" => sections |= 1 << SECTION_TCP,
-                "udp" => sections |= 1 << SECTION_UDP,
-                "icmp" => sections |= 1 << SECTION_ICMP,
                 "dev" => sections |= 1 << SECTION_DEV,
                 "ns" => sections |= 1 << SECTION_NS,
                 "meta" => sections |= 1 << SECTION_META,
                 "dataref" => sections |= 1 << SECTION_DATA_REF,
                 "gso" => sections |= 1 << SECTION_GSO,
-                "packet" => {
-                    warn!("Use of 'packet' in --skb-sections is depreacted (is now always set)")
+                "packet" | "eth" | "arp" | "ip" | "tcp" | "udp" | "icmp" => {
+                    warn!(
```

Comment:
does this worth (maybe along with packet, if makes sense) a note/update in the doc?
WDYT?

Reply:
Absolutely!